showpass/Showpass-E2E-tests
# TA-87 Ensure that guests are able to purchase tickets

**Description:**
When a user is not logged in, they should still have the ability to purchase tickets as a guest.
This script will be verifying that this scenario is still possible.

Steps:

- Create a new valid event
- Navigate to the web portal [Create Amazing Experiences](https://beta.showpass.com/) 
- DO NOT log in
- Find just created event and add tickets to cart
- Go through checkout as a guest and ensure the tickets can be purchased

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-87

**How to test:**
Run the following spec and ensure the tests pass cypress/e2e/events/barcodes.cy.js

**Test execution results:**

https://cloud.cypress.io/projects/e9axi7/branches/TA-87-checkout-as-guest/overview
